setup.py

```diff
@@ -2,17 +2,33 @@
 
 from distutils.core import setup
 
+#try:
+#    import pypandoc
+#    long_description = pypandoc.convert('README.md', 'rst')
+#except(IOError, ImportError):
+#    long_description = open('README.md').read()
+
```

Comment:
You can set description and long description like so https://github.com/bluedynamics/node/blob/master/setup.py#L8